systers_portal/templates/common/contact.html

```diff
@@ -0,0 +1,22 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% block content %}
+<!-- Facebook, Twitter, Linkedin, Github, Systers keeper: Rose Robinson -->
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>Contact Systers</h1>
+    <hr>
+    <p class="contact-text">
+      Systers is curated by the current <a href="mailto:systers-keeper@systers.org">Systers-keeper</a>, Rose Robinson.
+    </p>
+    <p class="contact-text">
+      You can also connect with Systers on our social networks.
+    </p>
+    <div class="container-fluid contact-social">
```

Comment:
Hmm, this bootstrap class `container-fluid` adds some padding to left and right. I think it doesn't look so good with this offset from the left. I assume the class can be safely removed or substituted with another CSS class.